index.html

```diff
@@ -198,6 +206,37 @@ <h1>Featured Speakers</h1>
           />
         </button>
       </div>
+      <!-- More arrow -->
     </section>
+    <footer>
+      
+      <div class="lastcontainer">
+        <div class="partner">
+          <h1>Partner</h1>
+          <p id="past"></p>
+        </div>
+        <ul class="browsers">
+          <li><a href="#" id="mozilla">mozilla</a></li>
+          <li><a href="#" id="google">Google</a></li>
+          <li><a href="#" id="naver">NAVER</a></li>
+        </ul>
+
+        <!-- technologies -->
+        <ul class="technologies">
+          <li><img src="./images/icons/github.svg" alt="github link" /></li>
+          <li><img src="./images/icons/bird.svg" alt="bird img" /></li>
+        </ul>
+      </div>
```

Comment:
- [ ] I can see you made a great effort implementing this footer to the desktop version. The partners design is still a bit different from the given design. To make it more similar, you should consider displaying them in a single row, instead of 2 rows. 
- [ ] When I checked the mobile version, I realized that the footer sections is visible there. To match the original design better, you should consider setting a `display: hidden`.

Your design:
![image](https://user-images.githubusercontent.com/66967274/147353143-17b144d6-100b-4569-a44d-3855e066525b.png)